remote/web/WEB-INF/web.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this web.xml required if there aren't any configuration changes specified?

Reply:
Not required, good catch! Deleted the file.